sample/SampleHost/Functions.cs

```diff
@@ -17,62 +19,111 @@ public class Functions
         private readonly ISampleServiceA _sampleServiceA;
         private readonly ISampleServiceB _sampleServiceB;
 
+        private static readonly Random _rand = new Random();
+
         public Functions(ISampleServiceA sampleServiceA, ISampleServiceB sampleServiceB)
         {
             _sampleServiceA = sampleServiceA;
             _sampleServiceB = sampleServiceB;
         }
 
-        [Singleton]
-        public void BlobTrigger(
-            [BlobTrigger("test")] string blob, ILogger logger)
+        public static async Task ConcurrencyTest_HighCpu([QueueTrigger("concurrency-work-items-1")] string message, ILogger log)
```

Comment:
I won't be checking in these test function changes. They're the functions I've been using for local dynamic concurrency testing

Reply:
I'm adding a new DynamicConcurrency E2E test suite that uses these functions and verifies concurrency management. While I have unit tests that also verify the various throttles, I agree it's beneficial to also have these tests.